iBudget/static/src/components/Login.vue

```diff
@@ -1,136 +1,175 @@
 <template>
-  <div class="content">
-      <div id="body">
-           <div id="form">
-              <form class="login" @submit.prevent="login" v-show="loginDisplay">
-                <h1>Login</h1>
-                <label>User name</label>
-                <br/>
-                <input required v-model="username" type="text" placeholder="Username"/>
-                <br/>
-                <label>Password</label>
-                <br/>
-                <input required v-model="password" type="password" placeholder="Password"/>
-                <hr/>
-                <b-button @click="login">Login</b-button>
-                <br/>
-                <br/>
-                <b-link @click="showRegister">Create a new account</b-link>
-                <hr/>
-                <b-button @click="google">Sign in With Google</b-button>
-              </form>
-
-             <form class="register" @submit.prevent="register" v-show="registerDisplay" >
-                <h1>Register</h1>
-                <label>User name</label>
-                <br/>
-                <input required v-model="username" type="text" placeholder="Username"/>
-                <br/>
-                <label>Password</label>
-                <br/>
-                <input required v-model="password" type="password" placeholder="Password"/>
-                <br/>
-                <label>Password Confirmation</label>
-                <br/>
-                <input required v-model="password" type="password" placeholder="Confirm Password"/>
-                <hr/>
-                <b-button type="submit" @click="registration">Register</b-button>
-               <br/>
-               <br/>
-               <b-link @click="showLogin">Already registered?</b-link>
-             </form>
-           </div>
-      </div>
-  </div>
+    <div class="content">
```

Comment:
Make a mark for yourself that all this `<br>` should be replaced with proper styles in future.

Reply:
I will do as u said in the future.Thank u